geomesa-convert/geomesa-convert-avro-schema-registry/src/test/scala/org/locationtech/geomesa/convert/avro/registry/AvroSchemaRegistryConverterTest.scala

```diff
@@ -17,7 +17,8 @@ import org.junit.runner.RunWith
 import org.locationtech.geomesa.convert2.SimpleFeatureConverter
 import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
 import org.locationtech.geomesa.utils.io.WithClose
-import org.mortbay.jetty.{Handler, Request, Server}
+import org.mortbay.jetty.{Connector, Handler, Request, Server}
```

Comment:
Why are there changes to this test?  Was it breaking on your machine?

Reply:
Yes, I happened to have another daemon running on port 8089 at the time which caused the test fail. I figured that I'd remove that landmine for everyone else.

Reply:
@elahrvivaz if you are happy, I'm happy.